packages/editor-ui/src/stores/npsSurvey.store.ts

```diff
@@ -135,19 +139,24 @@ export const useNpsSurveyStore = defineStore('npsSurvey', () => {
 			return;
 		}
 
-		const promptsData: IN8nPrompts = await getPromptsData(
-			settingsStore.settings.instanceId,
-			currentUserId.value ?? '',
-		);
+		try {
+			promptsData.value = await getPromptsData(
+				settingsStore.settings.instanceId,
+				currentUserId.value ?? '',
+			);
+		} catch (e) {
+			console.error('Failed to fetch prompts data');
+		}
```

Comment:
What happens if this fails?

Reply:
nothing, but maybe the nps survey will show. n8n api might be down, but it should not affect UX and no need to show notification (we make this call too often and user does not know about it).